src/test/scala/org/broadinstitute/dsde/agora/server/webservice/AgoraImportSpec.scala

```diff
@@ -67,24 +86,28 @@ class AgoraImportSpec extends ApiServiceSpec with FlatSpecLike{
     }
   }
 
-  "MethodsService" should "return a 400 when copying a method and specifying a WDL with an invalid import statement" in {
+  "MethodsService" should "return a 500 when copying a method and specifying a WDL with an invalid import statement" in {
     Post(copyUrl, copyPayload(testBadAgoraEntityInvalidWdlImportFormat.payload)) ~>
       methodsService.querySingleRoute ~> check {
-      assert(status == BadRequest)
+      assert(status == InternalServerError)
+      assert(body.asString contains "Failed to import workflow invalid_syntax_for_tool.:")
     }
   }
 
-  "MethodsService" should "return a 400 when copying a method and specifying a WDL with an import statement that references a non-existent method" in {
+  "MethodsService" should "return a 500 when copying a method and specifying a WDL with an import statement that references a non-existent method" in {
     Post(copyUrl, copyPayload(testBadAgoraEntityNonExistentWdlImportFormat.payload)) ~>
       methodsService.querySingleRoute ~> check {
-      assert(status == BadRequest)
+      assert(status == InternalServerError)
+      assert(body.asString contains "Failed to import workflow broad.non_existent_grep.1.:")
```

Comment:
Regarding these tests ... they cause a big stack trace in the test output, which is harmless but affects a developer's ability to quickly parse test results, AND we're throwing 500 when a 400 is more appropriate, since it's bad user input.

I suggest modifying the `handleException` method of `org.broadinstitute.dsde.agora.server.webservice.PerRequest` and adding: 

```
    case e: wdl4s.wdl.exception.ValidationException =>
      r.complete(BadRequest,AgoraException(e.getMessage, e.getCause, BadRequest))
```
with that case in place, we won't fall into the catchall `case e: Throwable`, which contains the logger statement that barfs stack traces into test output, and we'll throw a 400 instead of 500. Of course, some of these unit tests will need an update b/c they expect the 500.